includes/general.php

```diff
@@ -1052,7 +1052,17 @@ function pods_shortcode_run( $tags, $content = null ) {
 
 		return $pod->form( $tags['fields'], $tags['label'], $tags['thank_you'] );
 	} elseif ( ! empty( $tags['field'] ) ) {
-		if ( empty( $tags['helper'] ) ) {
+		if ( ! empty( $tags['template'] ) ) {
+			// @todo Single Pods query based on relationship field.
+			$return  = '';
+			$related = (array) $pod->field( $tags['field'], array( 'output' => 'pods', 'single' => false ) );
```

Comment:
* :no_entry_sign: Short array syntax must be used to define arrays
* :no_entry_sign: When a multi-item array uses associative keys, each value should start on a new line.

<sub>hash: 18997f6c3ffebb35482ff2e9ad51df3d</sub>